features/features.go

```diff
@@ -819,12 +819,20 @@ var (
 					enhancementPR("https://github.com/openshift/enhancements/pull/1779").
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
-					
+
 	FeatureGateAWSDedicatedHosts = newFeatureGate("AWSDedicatedHosts").
 					reportProblemsToJiraComponent("Installer").
 					contactPerson("faermanj").
 					productScope(ocpSpecific).
 					enhancementPR("https://github.com/openshift/enhancements/pull/1781").
 					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
+
+	FeatureGateVSphereMixedNodeEnv = newFeatureGate("VSphereMixedNodeEnv").
+					reportProblemsToJiraComponent("splat").
+					contactPerson("vr4manta").
+					productScope(ocpSpecific).
+					enhancementPR("https://github.com/openshift/enhancements/pull/1772").
```

Comment:
This EP is closed?

Reply:
I'll refresh it and create a new one.  This is the one we currently have.  It went stale.

Reply:
I have re-opened the PR and removed rotten label.  I'll be updating this doc this week with more of my findings of what we need to address.